libs/navigation/src/facades/navigation.facade.ts

```diff
@@ -25,10 +25,10 @@ export class DaffNavigationFacade implements DaffNavigationFacadeInterface {
    */
   errors$: Observable<string[]>;
 
-  constructor(private store: Store<NavigationReducersState>) {
-    this.tree$ = this.store.pipe(select(selectNavigationTree));
-    this.loading$ = this.store.pipe(select(selectNavigationLoading));
-    this.errors$ = this.store.pipe(select(selectNavigationErrors));
+  constructor(private store: Store<DaffNavigationReducersState<T>>) {
+    this.tree$ = this.store.pipe(select(getDaffNavigationSelectors<T>().selectNavigationTree));
```

Comment:
Can you move these calls into a const in the constructor, and then use the result?

Reply:
I made a commit. Is that what you mean?